### DISCUSSION TITLE
Is there FreeBSD support?

### DISCUSSION THREAD

**Discussion by unknowndevQwQ** [coder/coder#4373] [General] [answered]
My use case is to connect to a freebsd host via vscode-remote-ssh, but the current vscode-remote-ssh server does not yet support freebsd

**Comment by unknowndevQwQ** [comment]
@bpmct

**Comment by matifali** [comment, marked as answer]
Coder does support FreeBSD, Please try with our install script.
```curl -L https://coder.com/install.sh | sh```
